### PR TITLE
[APB-4877][GW] LLP assurance - additional content on NINO and Date of…

### PR DESCRIPTION
**Pull request by geoffreywatson**
… Birth pages

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -267,10 +267,12 @@ postcode.llp.title=Beth yw cod post cyfeiriad cofrestredig eich partneriaeth ate
 #Nino page
 nino.title=Beth yw’ch rhif Yswiriant Gwladol?
 nino.hint=Mae hwn ar eich cerdyn Yswiriant Gwladol, llythyr ynghylch budd-daliadau, slip cyflog neu P60. Er enghraifft, ‘QQ 12 34 56 C’.
+nino.hint-llp=
 
 #Date of Birth page
 date-of-birth.title=Beth yw’ch dyddiad geni?
 date-of-birth.hint=Er enghraifft, 31 3 1980
+date-of-birth.hint-llp=
```
Comment (sureshhmrc) [inline conf/messages.cy:275]: just add the default english text here - incase if we have to deploy the service before welsh gets ready ...
seing english is not worse than seeing empty text

File: app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/DateOfBirthController.scala [hunk 2]
```diff
@@ -66,39 +66,35 @@ class DateOfBirthController @Inject()(
...
   def submitDateOfBirthForm(): Action[AnyContent] = Action.async { implicit request =>
-    withSubscribingAgent { _ =>
-      dateOfBirthForm
-        .bindFromRequest()
-        .fold(
-          formWithErrors => Ok(dateOfBirthTemplate(formWithRefinedErrors(formWithErrors))),
-          validDob => {
-            sessionStoreService.fetchAgentSession.flatMap {
-              case Some(existingSession) =>
-                existingSession.nino match {
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/DateOfBirthController.scala:86]: is there any reason why you removed the nino check in your changes ?

Comment (geoffreywatson) [reply app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/DateOfBirthController.scala:86]: the presence of a nino is checked in `checkSessionStateAndBusinessType` -- if not present (for all businessTypes requiring nino) redirect to show nino page

File: conf/messages.cy (hunk 1 above)
Comment (geoffreywatson) [reply conf/messages.cy]: OK, I added the English text, thx

**Pull request merged**